src/routes/boot.ts

```diff
@@ -89,6 +98,7 @@ export default async function (fastify: FastifyInstance): Promise<void> {
         notifications: { unreadNotificationsCount },
         squads,
         features,
+        changelog: alerts.lastChangelog < new Date(lastChangelog),
```

Comment:
I forgot we have an alerts object. Should we put this inside the alerts object? Makes more sense I guess?

Reply:
I added the actual changelog date as the user object.
The evaluation always needs to happen seperate, not sure it makes sense to sync that to the database?

Reply:
We don't need to sync that to the db. I mean something like:
```
alerts: {
  ...alerts,
  lastChangelog: alerts.lastChangelog < new Date(lastChangelog),
}
```
Client doesn't really need the timestamp itself just the true/false.

Reply:
Ah right, yeah we can do something like that.
The update will always happen as a date anyway.

Reply:
Hmm, but userAlerts is actually a datetime in the database, then we're mixing up types which might not be great.
Maybe we can simply return alerts.changelog seperate?

Reply:
Ok, np 😁